### PR TITLE
Added content on README, Added Kenny Lin and Emma April on Contact..

### PR DESCRIPTION
**Pull request by ChuanKaiLin**
Yeah 👍

### CODE REVIEW

File: Contact.json [hunk 1]
```diff
@@ -17,6 +17,25 @@
...
+        {
+            "attributes": {
+                "type": "Contact",
+                "referenceId": "Emma"
+            },
+            "FirstName": "Emma",
+            "LastName": "April",
+            "Phone": "123456678"
         }
+
```
Comment (sungysang) [inline Contact.json]: Remove the newline

File: Contact.json [hunk 2]
```diff
@@ -17,6 +17,25 @@
...
+                "type": "Contact",
+                "referenceId": "Emma"
+            },
+            "FirstName": "Emma",
+            "LastName": "April",
+            "Phone": "123456678"
         }
+        
     ]
-}
```
Comment (sungysang) [inline Contact.json:40]: Add back last line

**Review by sungysang — COMMENTED** [review]
Formatting issues

File: README.md [hunk 3]
```diff
@@ -12,7 +17,12 @@ The `sfdx-project.json` file contains useful configuration information for your
...
 
+
+
 - [Salesforce Extensions Documentation](https://developer.salesforce.com/tools/vscode/)
 - [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 - [Salesforce DX Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_intro.htm)
 - [Salesforce CLI Command Reference](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference.htm)
+#Ths is Kenny!!!
+#Taiwan is good. 
+#Testing again
```
Comment (Julian88Tex) [inline README.md:28]: China is actually #1!!!!

**Pull request merged**